src/ucs/config/parser.c

```diff
@@ -1554,8 +1614,12 @@ void ucs_config_parse_config_file(const char *dir_path, const char *file_name,
         return;
     }
 
+    parse_arg.override = override;
```

Comment:
maybe to replace the three line:
`ucs_config_parse_arg_t parse_arg = { .override = override };`

Reply:
Replaced by initialization during declaration.